test/helper.js

```diff
@@ -125,4 +146,22 @@ mocha.after(async () => {
   if (webServer) {
     await webServer.terminate();
   }
+  if (cfg.azure && cfg.azure.accountName && cfg.azure.accountKey) {
+    const blobService = new azure.Blob({
+      accountId: cfg.azure.accountName,
+      accountKey: cfg.azure.accountKey,
+    });
+    try {
+      await blobService.deleteContainer(helper.containerName);
+    } catch (e) {
+      if (e.code !== 'ResourceNotFound') {
+        throw e;
+      }
+      // already deleted, so nothing to do
+      // NOTE: really, this doesn't work -- the container doesn't register as existing
+      // before the tests are complete, so we "leak" containers despite this effort to
+      // clean them up.
```

Comment:
This bums me out, but seems to be how Azure works.  I noticed that deleting blobs has some synchronization issues, too: a DELETE followed immediately by a GET will return the deleted data.